tests/components/ondilo_ico/test_config_flow.py

```diff
@@ -0,0 +1,88 @@
+"""Test the Ondilo ICO config flow."""
+from unittest.mock import patch
+
+from homeassistant import config_entries, data_entry_flow, setup
+from homeassistant.components.ondilo_ico import config_flow
+from homeassistant.components.ondilo_ico.const import (
+    DOMAIN,
+    OAUTH2_AUTHORIZE,
+    OAUTH2_CLIENTID,
+    OAUTH2_CLIENTSECRET,
+    OAUTH2_TOKEN,
+)
+from homeassistant.const import CONF_CLIENT_ID, CONF_CLIENT_SECRET
+from homeassistant.helpers import config_entry_oauth2_flow
+
+from tests.common import MockConfigEntry
+
+CLIENT_ID = OAUTH2_CLIENTID
+CLIENT_SECRET = OAUTH2_CLIENTSECRET
+
+
+async def test_abort_if_existing_entry(hass):
+    """Check flow abort when an entry already exist."""
+    MockConfigEntry(domain=DOMAIN).add_to_hass(hass)
+
+    flow = config_flow.OAuth2FlowHandler()
```

Comment:
Don't create the flow directly. Start the test with `hass.config_entries.flow.async_init`.

Reply:
Done in PR https://github.com/home-assistant/core/pull/44837